tomo_challenge/metrics.py

```diff
@@ -154,3 +245,141 @@ def plot_distributions(z, tomo_bin, filename, nominal_edges=None):
 
     plt.savefig(filename)
     plt.close()
+
+
+def make_sacc(tomo_bin, z, what, mu, C):
+    # Basic numbers
+    nbin = int(tomo_bin.max()) + 1
+    tracertype = get_tracertype(nbin,what)
+    ntot = nbin*2 if what=='3x2' else nbin
+    
+    
+    z_mid, n_of_z = get_n_of_z(tomo_bin, z)
+    npair = (ntot * (ntot + 1)) // 2
+
+    # Must be the same as above
+    ell = ell_binning()
+    n_ell = len(ell)
+    # Just EE for now
+    EE = sacc.data_types.standard_types.galaxy_shear_cl_ee
+    GG = sacc.data_types.standard_types.galaxy_density_cl
+    GE = sacc.data_types.standard_types.galaxy_shearDensity_cl_e
+    # Start with empty sacc.
+    S = sacc.Sacc()
+
+    # Add all the tracers
+    for ci in range(ntot):
+        i = ci %nbin
+        if tracertype[ci]=='g':
+            S.add_tracer("NZ", f'lens_{i}', z_mid, n_of_z[i])
+        elif tracertype[ci]=='w':
+            S.add_tracer("NZ", f'source_{i}', z_mid, n_of_z[i])
+        else:
+            raise NotImplemented
```

Comment:
This should be `NotImplementedError` 
(`NotImplemented` is for overriding mathematical operators in classes)